lmdeploy/cli/entrypoint.py

```diff
@@ -10,8 +11,8 @@
 def run():
     """The entry point of running LMDeploy CLI."""
     args = sys.argv[1:]
-    is_deprecated_chat_cli = args[0] == 'chat' and (args[1]
-                                                    in ['torch', 'turbomind'])
+    is_deprecated_chat_cli = (len(args) > 0 and args[0] == 'chat') and \
```

Comment:
My mistake. To be more precisely, it could be `len(args) > 1 and args[0] ==  'chat' and args[1] in ['torch', 'turbomind']`